canonical/canonical-design
# Wd 16711 fix static resources icons favicons shift logos to cdn

## Done
- Replaced static images with server image.

## QA
- Run dotrun locally
- Visit all the pages and see if images are present and loading up (apart from blog sections/page).

## Fixes

[WD-16711](https://warthogs.atlassian.net/browse/WD-16711)

[WD-16711]: https://warthogs.atlassian.net/browse/WD-16711?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
